wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -422,7 +422,7 @@ public final class MLSService: MLSServiceInterface {
     /// - Throws:
     ///   - MLSGroupCreationError if the group could not be created.
 
-    public func createGroup(for groupID: MLSGroupID) throws {
+    public func createGroup(for groupID: MLSGroupID) async throws {
         logger.info("creating group for id: \(groupID.safeForLoggingDescription)")
```

Comment:
I think we're missing ` await fetchBackendPublicKeys()` here check ` // FIXME: [jacob] fetch on demand when creating group`

Reply:
Will be done in PR https://github.com/wireapp/wire-ios/pull/796